src/test/regress/sql/pg_dump.sql

```diff
@@ -54,11 +76,34 @@ DROP SCHEMA dumper CASCADE;
 -- redistribute the schema
 SELECT create_distributed_table('data', 'key');
 SELECT create_distributed_table('"weird.table"', 'key,');
+SELECT create_distributed_table('dist_columnar', 'i');
 
 -- check the table contents
 COPY data (value) TO STDOUT WITH (format csv, force_quote *);
 COPY dumper."weird.table" ("data.jsonb", "?empty(") TO STDOUT WITH (format csv, force_quote ("?empty("), null 'null', header true);
 
+-- If server supports table access methods, check to be sure that the
+-- recreated table is still columnar. Otherwise, just return true.
+\if :have_columnar
```

Comment:
this should be `\if :have_table_am`

Reply:
You're right, but then I realize that different branches are taken and it doesn't really work for both versions with the same exact output. Any suggestions?

Reply:
Good catch. I think I found a solution that works for v11 and v12.